src/brogue/Recordings.c

```diff
@@ -726,6 +726,205 @@ void pausePlayback() {
     }
 }
 
+// Pass in defaultPath (the file name WITHOUT suffix), and the suffix.
+// Get back either defaultPath, or "defaultPath N",
+// where N is the lowest counting number that doesn't collide with an existing file.
+void getAvailableFilePath(char *returnPath, const char *defaultPath, const char *suffix) {
```

Comment:
Looks like a bunch of code has been moved to earlier in the file. Is there a reason for this?

Reply:
SwitchToPlaying wasn't defined in Rogue.h, but I somehow missed that. I moved every function it depended on before executePlaybackInput, but I'll change that back.

Reply:
Also, as for attaching this to wizard mode, wizard mode goes out of sync turn 0 when playing any save file/recording, so I would have to make changes to that as well.